homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -638,7 +638,7 @@ def __init__(self, name, device, entry, unique_id):
     @property
     def supported_features(self):
         """Flag supported features."""
-        return SUPPORT_SET_SPEED
+        return SUPPORT_PRESET_MODE
```

Comment:
If we just add preset mode support and keep set speed support, the old set speed service should still work for users to set presets. Ie it will not be a breaking change.

Please check this.

Reply:
I've removed the feature flag to disable the slider at the frontend.

Reply:
Do you agree with the breaking change? I will update the description of the PR in this case.

Reply:
Ok.